apps/frontend/utils/tableFormatting.tsx

```diff
@@ -48,11 +74,14 @@ const getLocationColumnSetup = (
 ): GridColDef => {
   return {
     field,
-    width,
+    width: width + sortIconWidth,
```

Comment:
Since we are lacking real estate, I would be in favor of just hiding the sortIcon completely but still make the column sort itself on click. wdyt?